content/patterns/button/examples/button.html

```diff
@@ -139,8 +140,8 @@ <h2 id="rps_label">Role, Property, State, and Tabindex Attributes</h2>
               <td><code>div</code>, <code>a</code></td>
               <td>
                 <ul>
-                  <li>Includes the element in the tab sequence.</li>
-                  <li>Needed on the <code>a</code> element because it does not have a <code>href</code> attribute.</li>
+                  <li>要素をタブの順序に含めます。</li>
+                  <li><code>href</code> 属性を持っていない <code>a</code> 要素に必要です。</li>
```

Comment:
こちらも言い切りの形に。
2つめのliは、bacause が訳出されていなかったので「ため」を足しました。

```suggestion
                  <li>要素をタブ順序に含める。</li>
                  <li><code>href</code> 属性を持たないため <code>a</code> 要素に必要。</li>
```